lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -28,6 +28,7 @@ def initialize(input, suspended_action:, id: nil)
         @passphrase = action_input['secrets']['key_passphrase']
         @execution_timeout_interval = action_input[:execution_timeout_interval]
         @cleanup_working_dirs = action_input.fetch(:cleanup_working_dirs, true)
+        prune_known_hosts_on_first_execution(input.values, logger)
```

Comment:
`logger` is a reader method granting access to `@logger` instance variable. Why pass it into the method as an argument?

Reply:
You're right, I missed that when I added my changes.